app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -104,8 +108,28 @@ def parse_vm(vm)
       :operating_system => {
         :product_name => guest_os,
       },
+
+      :orchestration_stack => stack,
     }
 
     return uid, new_result
   end
+
+  def parse_stack(vapp)
+    # vApp and VM status mapping is identical, see link below
+    # http://pubs.vmware.com/vcd-80/topic/com.vmware.ICbase/PDF/vcloud_sp_api_guide_90.pdf - page 421
+    status   = ManageIQ::Providers::Vmware::CloudManager::Vm::calculate_power_state(vapp.human_status)
```

Comment:
I've discussed with @cankarm - the mapping should take place in fog library, not here in manageiq. Is it okay if we use whatever fog returns, and will later update fog?